marklogic-data-hub/src/test/java/com/marklogic/hub/step/impl/RunDefaultIngestionTest.java

```diff
@@ -0,0 +1,32 @@
+package com.marklogic.hub.step.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.datamovement.WriteBatcher;
+import com.marklogic.client.document.ServerTransform;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.Format;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.io.StringHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+public class RunDefaultIngestionTest extends AbstractHubCoreTest {
+
+    @Test
+    void runDefaultIngestion(){
+        ServerTransform serverTransform = new ServerTransform("mlRunIngest");
+        DatabaseClient stagingClient = getHubClient().getStagingClient();
+        WriteBatcher writeBatcher = stagingClient.newDataMovementManager().newWriteBatcher()
+            .withDefaultMetadata(new DocumentMetadataHandle().withPermission("data-hub-common", DocumentMetadataHandle.Capability.READ))
+            .withTransform(serverTransform);
+        writeBatcher.add("/test.json", new StringHandle("{\"testing\": \"value\"}").withFormat(Format.JSON));
+        writeBatcher.flushAndWait();
+        JsonNode docNode = stagingClient.newJSONDocumentManager().read("/test.json").next().getContent(new JacksonHandle()).get();
+        Assertions.assertNotNull(docNode);
+        Assertions.assertEquals("value", docNode.get("envelope").get("instance").get("testing").asText());
```

Comment:
Can you add an assertion on the Job document that gets created (which unfortunately is left in an unfinished state) on flowName = default-ingestion? I'm guessing that's not tested anywhere, so would be good to test here. There's no indication that the first step was run, although that's implied by the flow finishing successfully.